lib/composites/menu/events/main.js

```diff
@@ -36,7 +36,7 @@ module.exports = (elements) => {
       elements.trigger.addEventListener('click', onTriggerClick);
     }
 
-    if (hasTopBarDropdown) {
+    if (!hasTopBarDropdown) {
```

Comment:
maybe we should name this `isWithinDropdown` or something similar to make this code more clear. 
P.S. I know the naming of this is somewhat my fault since it came out of that code block I wrote in [this comment](https://github.com/dequelabs/pattern-library/pull/49#discussion_r163045588) on the previous PR

Reply:
`isWithinDropdown` makes sense. Think we just lost track of the name in the mix of changing other things.